runners/wikiConsole/templates/npmInstall.asciidoc

```diff
@@ -0,0 +1,19 @@
+Installation of packages with npm install
```

Comment:
I would find it better if you write the name of the package here if a package is given and write 'Install dependencies' if the user should install the packages from the package.json file